xep-0402.xml

```diff
@@ -102,25 +115,42 @@
 <conference xmlns=']]>&namespace;<![CDATA['
             name='Council of Oberon'
             autojoin='true'>
-    <nick>Puck</nick>
+  <nick>Puck</nick>
 </conference>
 ]]></example>
         <p>This bookmark would be displayed as 'Council of Oberon' and, if activated, would attempt to join the conference room 'council@conference.underhill.org' with nickname 'Puck'.</p>
         <p>Note that a bookmark item MUST contain only one conference room.</p>
-        <p>Note also that a conference element has no truly mandatory attributes or child elements, though a name SHOULD be given. Thus the following is legal:</p>
+        <p>Note also that a conference element has no truly mandatory attributes or child elements, thus the following is legal:</p>
         <example caption='Minimalist conference element'><![CDATA[
 <conference xmlns=']]>&namespace;<![CDATA['/>
 ]]></example>
     </section1>
 
-    <section1 topic='Examples' anchor='examples'>
+    <section1 topic='Workflow' anchor='workflow'>
+
+        <section2 topic='Registering to notifications' anchor='enable-updates'>
+            <p>A client interested in bookmarks SHOULD include the 'urn:xmpp:bookmarks:0+notify' feature in its &xep0115;, as per &xep0163;, so that it receives notifications for updates done by other clients of the user, and react accordingly.  The actual notifications are explained in the <link url='#notifications'>Bookmark Notifications</link> section of this specification.</p>
+
+            <example caption='Client replies with +notify to a XEP-0030 query from its server'><![CDATA[
```

Comment:
Should XEP-0030 be linkified? e.g. `&xep0030;`

Reply:
Using literal XML tags inside of an attribute is invalid in XML, so we can’t add any markup here.